src/main/java/frc/robot/cmdGroup/SpinUpAndShoot.java

```diff
@@ -10,7 +10,7 @@
 public class SpinUpAndShoot extends SequentialCommandGroup {
   public SpinUpAndShoot(ShooterSubsystem shooter, BooleanSupplier slowShoot) {
     addCommands(
-        new SpinUp(shooter, slowShoot),
+        new SpinUp(shooter, Constants.Shooter.SHOOT_SPEED, slowShoot),
```

Comment:
"resolving the merge conflict" means making this change in SUAS in the new place, too.